.github/workflows/integration_test.yml

```diff
@@ -28,14 +25,6 @@ jobs:
         with:
           go-version: ${{env.GO_VERSION}}
 
-      - name: build manifest
-        run: make build-manifest OPTS="CI_DEPLOY"
-        env:
-          CLUSTER_PROVIDER: ${{matrix.cluster_provider}}
-          IMAGE_REPO: "localhost:5001"
-          IMAGE_TAG: "devel"
-          CTR_CMD: docker
```

Comment:
<img width="1197" alt="image" src="https://github.com/sustainable-computing-io/kepler/assets/3005132/b1385710-7b0c-4d4b-8ca8-efaab69172e3">

I see there a difference in the generation based on cluster-provider. Is that okay?

Reply:
There is also a difference in the `exporter` kustomization.yaml 

See: 
<img width="1215" alt="image" src="https://github.com/sustainable-computing-io/kepler/assets/3005132/7b57244d-3b10-4ed0-b654-a30387d719cc">

Reply:
This affects only when building a manifest so I think the `build-kepler` should be fine.